sources/web/datalab/auth.ts

```diff
@@ -146,8 +146,8 @@ function persistCredentials(tokens: any): string {
 }
 
 export function isSignedIn(): boolean {
-  // If not local, then we should have service account available so consider that signed in.
-  return (process.env.DATALAB_ENV != 'local' || fs.existsSync(appCredFile));
+  var gcloudAccount:string = getGcloudAccount();
```

Comment:
I tried the command in getGcloudAccount() ("gcloud config list --format "value(core.account)") when I already signed out, and it still returns my user id. Perhaps core.account does not reflect the sign-in status?

On the other hand, 'gcloud auth list --format="value(account)"' seems working for me.